ui/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Maintenance tracker</title>
+  <link href="style.css" rel="stylesheet">
+  <!--google fonts-->
+  <link href="https://fonts.googleapis.com/css?family=Skranji" rel="stylesheet">
+  <!--font awesome -->
+  <script src="fontawesome-all.js"></script>
+</head>
+
+<body>
+  <!--header-section-begin-->
+  <header class="header">
+    <nav>
+      <div class="nav-container">
+        <div class="btn">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </div>
+      </div>
+      <ul>
+        <div class="logo">Maintenance Tracker</div>
+        <li>
+          <a href="#">home</a>
+        </li>
+        <li>
+          <a href="#about-section">about</a>
+        </li>
+        <li>
+          <a href="signin.html">sign in</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="banner">
+      <div class="banner-center">
+        <h1>Maintenance Tracker</h1>
+        <h3>Repairing one day at a time</h3>
+        <div>
+          <a class="button" href="signin.html">Sign In</a>
+          <a class="button" href="signup.html">Sign Up</a>
+        </div>
+        <div class="banner-icons">
+          <a href="">
+            <i class="fab fa-facebook"></i>
+          </a>
+          <a href="">
+            <i class="fab fa-instagram"></i>
+          </a>
+        </div>
+      </div>
+    </div>
+  </header>
+  <!--header-section-end-->
+
+  <!-- About Section -->
+  <section class="about" id="about-section">
+    <div class="about-center">
+      <div class="about-heading">
+        <h2>About</h2>
+      </div>
+      <article class="about-article">
+        <div class="about-icon">
+            <i class="fas fa-wrench"></i>
+        </div>
+        <div class="about-text">
+          <h3>Quick Maintenance</h3>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
```

Comment:
You might do well to add more meaningful text here. This would help you to be able to finish this task and move to the next instead of going back and forth.

Reply:
Alright, I will implement that